styles.css

```diff
@@ -0,0 +1,1053 @@
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+}
+
+footer .top-footer nav .icons .icon {
+  font-size: 130%;
+}
+
+footer .top-footer nav .icons .rss {
+  display: none;
+}
+
+footer .top-footer nav .other-links {
+  width: 85%;
+  display: flex;
+  justify-content: space-around;
+  margin: auto;
+  font-weight: 600;
+  font-size: 95%;
+}
+
+footer .top-footer nav .icons li a .fab,
+footer .top-footer nav .icons li a .fas {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  padding: 8px;
+  border: 2px solid #333;
+  background-color: #333;
+  color: #fff;
+  width: 40px;
+  height: 40px;
+}
+
+footer .top-footer nav .icons li a:hover .fab,
+footer .top-footer nav .icons li a:hover .fas {
+  background-color: #131313;
+  border: 2px solid #333;
+}
+
+footer .top-footer nav .other-links li .links {
+  color: gray;
+}
+
+footer .top-footer nav .other-links li .links:hover {
+  color: #fff;
+}
+
+footer .bottom-footer {
+  height: 122px;
+  background-color: #000;
+  text-align: center;
+  color: gray;
+  font-size: 82%;
+  padding: 32px 0;
+}
+
+footer .bottom-footer .heart {
+  color: #f42;
+}
+
+footer .bottom-footer .footer-img {
+  display: inline-block;
+  height: 9px;
+}
+
+/* ====== Media Queries  ====== */
+
+/* Small devices (portrait tablets and large phones, 600px and up) */
+@media only screen and (min-width: 600px) {
+  header nav .bottom-nav #ad-link {
+    font-size: 80%;
+  }
+
+  .most-popular .popular .lg-img {
+    height: 420px;
+  }
+
+  footer .top-footer nav .icons .rss {
+    display: inline-block;
+  }
+
+  footer .top-footer nav {
+    padding: 0;
+  }
+}
+
+/* Medium devices (landscape tablets, 768px and up) */
+@media only screen and (min-width: 768px) {
+  header {
+    height: 110px;
+  }
+
+  header nav .extra-links {
+    display: block;
+    height: 32px;
+    background-color: #000;
+    line-height: 32px;
+    font-size: 88%;
+    padding: 0 7px 0 120px;
+  }
+
+  header nav .extra-links li {
+    display: inline;
+    padding-right: 3px;
+  }
+
+  header nav .extra-links li a {
+    color: gray;
+  }
+
+  header nav .top-nav .middle-nav a {
+    color: gray;
+  }
+
+  header nav .extra-links li a:hover {
+    color: #fff;
+  }
+
+  header nav .top-nav .middle-nav a:hover {
+    color: #000;
+  }
+
+  header nav .extra-links #news-link a {
+    color: #fff;
+  }
+
+  header nav .extra-links li:nth-last-child(-n+4) {
+    float: right;
+    padding: 0 3px;
+  }
+
+  header nav .top-nav div {
+    position: relative;
+  }
+
+  header nav .top-nav div .logo {
+    display: none;
+  }
+
+  header nav .top-nav div .logo2 {
+    display: block;
+    position: absolute;
+    top: -32px;
+    height: 80px;
+    max-width: 106px;
+  }
+
+  header nav .bottom-nav {
+    height: 30px;
+    line-height: 30px;
+    text-align: left;
+    padding-left: 120px;
+  }
+
+  header nav .bottom-nav #ad-link {
+    line-height: 30px;
+    text-align: left;
+  }
```

Comment:
- [ ]  No specification needed, especially when you are using an ID!!! But please try to avoid the use of them as much as you can. It's not considered a good practice.